bnd-maven plugin: delete existing MANIFEST.MF from previous build (#6222)
bndtools/bnd: maven-plugins/bnd-maven-plugin/README.md

| `bndfile`               | File path to a bnd file containing bnd instructions for this project. The file path can be either absolute or relative to the project directory. _Defaults to `bnd.bnd`_.                                                                                                         |
| `bnd`                   | Bnd instructions for this project specified directly in the pom file. This is generally be done using a `<![CDATA[  ]]>` section. If the projects has a `bndfile` configuration property or a file in the default location `bnd.bnd`, then this configuration element is ignored. |
| `manifestPath`          | Specify the path to store the generated manifest file. _Defaults to `${project.build.outputDirectory}/META-INF/MANIFEST.MF`._                                                                                                                                                     |

> | `deleteExistingManifest`          | Whether to delete an existing file at `manifestPath`. Defaults to `false`

Reviewer: ```suggestion
| `deleteExistingManifest`          | Whether to delete an existing file at `manifestPath` before execution. _Defaults to `false`._
```

> | `deleteExistingManifest`          | Whether to delete an existing `META-INF/MANIFEST.MF` inside the `classesDir` directory. Defaults to `false`

Reviewer: ```suggestion
| `deleteExistingManifest`          | Whether to delete an existing file at `manifestPath`. Defaults to `false`
```

| `skip`                  | Skip the goal. _Defaults to `false`._ Override with property `bnd-tests.skip` or `maven.test.skip`.                                                                                                                                                                               |
| `skipIfEmpty`           | Skip processing if `includeClassesDir` is `true` and the `classesDir` is empty. _Defaults to `false`._ Override with property `bnd.skipIfEmpty`.                                                                                                                                  |
| `manifestPath`          | Specify the path to store the generated manifest file. _Defaults to `${project.build.testOutputDirectory}/META-INF/MANIFEST.MF`._                                                                                                                                                 |

> | `deleteExistingManifest`          | Whether to delete an existing `META-INF/MANIFEST.MF` inside the `classesDir` directory. Defaults to `false`

Reviewer: ```suggestion
| `deleteExistingManifest`          | Whether to delete an existing file at `manifestPath`. Defaults to `false`
```
